spec/lucky_cli/task_spec.cr

```diff
@@ -39,4 +39,49 @@ describe LuckyCli::Task do
       io.to_s.chomp.should eq "Custom help message"
     end
   end
```

Comment:
This will be tricky with the positional_arg. The others pass description to the built-in option parser, but since the positional has to be hand rolled, there's no place the description is actually used. 

My thought here is make a constant and push all of the data in to that, then iterate over this to display our own custom arg menu deal. This means that I need to know on all of them if there's a shortcut or not, which type it is, maybe the format? So this ends up looking like:

```crystal
{long: "--model_name=NAME", short: "-m NAME", format: "/\w+/", desc: "The name of the model"}
{long: "--force", short: "", format: "", desc: "Force this to work"}
{long: "Position 0", short: "", format: "/\w+/", desc: "First positional arg thing"}
{long: "Position 1..-1", short: "", format: "/\w+/", desc: "First positional arg thing"}
```

I don't really know how to best show the format for the positional, but they should all be the same so then I can do something like

```
CLI_ARG_HELP_OPTS.each do |data|
  str << data[:long]
  str << data[:short] if data[:short]
  str << "some sort of padding...." data[:desc]
  str << "(Format: #{data[:format]})"
end
```

Reply:
Hmm seems pretty complex. If you want to do that then I’m down but I’m also ok shipping as is. This is so awesome that I wouldn’t want to hold it up even longer haha. 

So up totally up to you!